environments/21.08/spack.yaml

```diff
@@ -0,0 +1,327 @@
+spack:
+
+  view: false
+  concretization: separately
+
+  packages:
+    all:
+      compiler:
+        - gcc@9.3.0
+      providers:
+        blas:
+          - openblas
+        mpi:
+          - mpich
+      target:
+        - x86_64
+      variants: +mpi
+    autoconf:
+      version:
```

Comment:
same thing with these `version` makes it easier to read and avoids issue with indentation issue since YAML is picky with how list are written

so write this as follows 

```
autoconf:
  version: [2.69]
```

Please make the change across all specs it will also cut down on number of lines and make it easier to read

Reply:
It is perfectly readable as it is. If you would like to have these changes, please feel free.